src/edu/csus/ecs/pc2/ui/ContestClockDisplay.java

```diff
@@ -48,6 +48,14 @@ public class ContestClockDisplay implements ActionListener, UIPlugin {
     private static final long serialVersionUID = 8137635697344335832L;
 
     private static final String NO_SCHEDULE_START_TIME = "No scheduled start";
+    private static final String NO_SCHEDULE_START_HINT = "Still no scheduled start";
```

Comment:
I feel like the word `SCHEDULE` in these constants should be `SCHEDULED`.
(And yes, I'm aware that one of them was pre-existing...)

Reply:
Fixed in next push.